src/components/common/Actions.vue

```diff
@@ -214,6 +216,18 @@ export default class Actions extends Mixins(CommonMixin, DateMixin, FilingTempla
     // Prompt Step validations
     this.setValidateSteps(true)
 
+    // If we're filing an amalgamation, we need to re-fetch the business table data on file and pay.
+    // We do that in case one of the businesses became invalid (e.g. historical) while the draft is open.
+    if (this.getFilingType === FilingTypes.AMALGAMATION_APPLICATION) {
```

Comment:
I moved the function here before the `isFilingValid` getter. To keep things as clean as possible, since the isFilingValid relies on the table being valid, the re-fetching is no working how it's supposed to.

Reply:
Small demo: 

https://github.com/bcgov/business-create-ui/assets/122301442/53abed75-0ac2-4f99-a19c-dce8c2e5d0ba